### PR TITLE
Nathan Karasch added to members page.

### PR DESCRIPTION
**Pull request by KrashLeviathan**
### Problem / Feature
Too many Nic Cages.
Issue #51

### Solution
Added myself to the members page.

### What To Test
It works.

### Reviewers

@JackRometty    

FYI: @KrashLeviathan @csteamengine @regalcat

### CODE REVIEW

**Comment by KrashLeviathan** [comment]
@JackRometty or @csteamengine could you look at this one and give it a +1?

**Comment by JackRometty** [comment]
looks good to me! do you just want me to comment after checking through the file or is there an official "reviewed" button?

**Comment by KrashLeviathan** [comment]
Check out the [Contributing](https://github.com/ISU-WebDevClub/club-website/blob/master/CONTRIBUTING.md) doc under the section "Code Review +1 Template". Some of the stuff won't apply (like testing at the moment) so you can just take it out, but you're basically making sure everything works and nothing got broken in the process. It's not a template we have to be super strict about, it just helps reviewers cover the bases when reviewing someone else's code.

File: members/members.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "members": {
-    "Nicolas Cage": {
-      "image": "http://www.placecage.com/g/200/200",
-      "description": "Mustachio extraordinaire"
+    "Nathan Karasch": {
+      "image": "../_images/members/Karasch-Nathan-md.jpg",
```
Comment (jtjudge) [inline members/members.json]: This is a line

**Pull request merged**